tools/crm_resource_print.c

```diff
@@ -33,19 +33,18 @@ cli_resource_print_cts_constraints(pcmk__output_t *out, pe_working_set_t * data_
             continue;
         }
 
-        if (pcmk__str_eq(XML_CONS_TAG_RSC_DEPEND, crm_element_name(xml_obj), pcmk__str_casei)) {
-            printf("Constraint %s %s %s %s %s %s %s\n",
-                   crm_element_name(xml_obj),
-                   cons_string(crm_element_value(xml_obj, XML_ATTR_ID)),
-                   cons_string(crm_element_value(xml_obj, XML_COLOC_ATTR_SOURCE)),
-                   cons_string(crm_element_value(xml_obj, XML_COLOC_ATTR_TARGET)),
-                   cons_string(crm_element_value(xml_obj, XML_RULE_ATTR_SCORE)),
-                   cons_string(crm_element_value(xml_obj, XML_COLOC_ATTR_SOURCE_ROLE)),
-                   cons_string(crm_element_value(xml_obj, XML_COLOC_ATTR_TARGET_ROLE)));
-
-        } else if (pcmk__str_eq(XML_CONS_TAG_RSC_LOCATION, crm_element_name(xml_obj), pcmk__str_casei)) {
-            /* unpack_location(xml_obj, data_set); */
+        if (!pcmk__str_eq(XML_CONS_TAG_RSC_DEPEND, crm_element_name(xml_obj), pcmk__str_casei)) {
+            continue;
         }
+
+        out->info(out, "Constraint %s %s %s %s %s %s %s",
+                  crm_element_name(xml_obj),
+                  cons_string(crm_element_value(xml_obj, XML_ATTR_ID)),
+                  cons_string(crm_element_value(xml_obj, XML_COLOC_ATTR_SOURCE)),
+                  cons_string(crm_element_value(xml_obj, XML_COLOC_ATTR_TARGET)),
+                  cons_string(crm_element_value(xml_obj, XML_RULE_ATTR_SCORE)),
+                  cons_string(crm_element_value(xml_obj, XML_COLOC_ATTR_SOURCE_ROLE)),
+                  cons_string(crm_element_value(xml_obj, XML_COLOC_ATTR_TARGET_ROLE)));
```

Comment:
Actually now that tools are gaining xml output it makes sense for cts to use and parse that. But that should be its own project (one of the "would be nice" ones that are always done "next year" ...)

Reply:
I could just drop this patch and make doing that conversion the next thing I work on.

Reply:
No, this is better for now. Maybe at some point, if it becomes a problem or we want a new capability.